### PR TITLE
Fix database access on main thread that caused a crash

### PR DESCRIPTION
**Pull request by CrisBarreiro**
Task/Issue URL: https://app.asana.com/0/1202552961248957/1208294040908744/f

### Steps to test this PR

_Feature 1_
- [x] Open https://www.disneyaulani.com/ 
- [x] Check app doesn't crash and works normally
- [x] Open maps.google.com and click on the location button
- [x] Check app doesn't crash and works normally

### CODE REVIEW

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt [hunk 1]
```diff
@@ -3224,21 +3226,23 @@ class BrowserTabViewModel @Inject constructor(
         }
     }
 
-    private fun permissionsQuery(
+    private suspend fun permissionsQuery(
```
Comment (0nko) [inline app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt:3229]: You’re launching a new coroutine inside this function, so it doesn’t need to be a suspended function. Also, the call to `permissionQuery()` is from within _another_ coroutine launcher (which is also the same dispatcher as here). I think you want to use one or the other. WDYT?

**Review by 0nko — COMMENTED** [review]
The app behaves as expected but I left a comment in the code because I think some coroutine launchers are redundant.

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt (hunk 1 above)
Comment (CrisBarreiro) [reply app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt]: Fixed, I missed that on my second commit, thanks!

**Review by 0nko — APPROVED** [review]
Nice work! :shipit:

**Pull request merged**